.eslintignore

```diff
@@ -1,3 +1,14 @@
-examples/js/GUI/dat.gui
-src/ThreeExtended/*
-examples/libs/*
+examples/libs/
+examples/layers/
+src/ThreeExtended/
+test/hooks.js
+dist/
+node_modules/
+docs/out/
+docs/static/
+docs/tutorials/
+docs/tmpl/
+test/data/
```

Comment:
What's with the weird things ? Doesn't eslint also look at `.gitignore` ?

Reply:
> Doesn't eslint also look at .gitignore

I don't think so

Reply:
It seems to have changed a bit (see https://eslint.org/docs/user-guide/migrating-to-7.0.0#default-ignore-patterns-have-changed), but ok with leaving this like it is